index.js

```diff
@@ -155,23 +147,35 @@ function createQuestions(config) {
  * Format the git commit message from given answers.
  *
  * @param {Object} answers Answers provide by `inquier.js`
+ * @param {Object} config Result of the `getConfig` returned promise
  * @return {String} Formated git commit message
  */
-function format(answers) {
+
+function format(answers, config) {
```

Comment:
For clarity, I think we should rename this function:
```suggestion
function formatCommitMessage(answers, config) {
```